src/javascript/static/pages/regulation.js

```diff
@@ -21,6 +24,12 @@ const Regulation = (() => {
                     relocateLinks();
                 }
             });
+
+            BinarySocket.wait('website_status').then(() => {
+                if (hasCFD()) {
+                    document.querySelectorAll('#cfd_fillbox').remove();
```

Comment:
if it's id there should be only a single element, so just getElementByID works (note that we have a custom function for that)

Reply:
Addressed in [#611f3d3](https://github.com/binary-com/binary-static/pull/4929/commits/611f3d3cecf7c31483d6e0bdcddaf510a3729b3c)

Reply:
Thanks. Addressed in [#6341808](https://github.com/binary-com/binary-static/pull/4929/commits/634180883314fe4bcb83e7ed513053538fb2aaf3).